drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -4,10 +4,12 @@
 <div class="xblock--drag-and-drop--editor editor-with-buttons">
     {{ js_templates|safe }}
 
-    <section class="drag-builder">
-        <div class="tab feedback-tab">
-
-            <section class="tab-content">
+    <div class="drag-builder">
+        <section class="tab feedback-tab">
+            <header class="tab-header">
+                <h3>{% trans "Basic Settings" %}</h3>
```

Comment:
Unfortunately the heading levels in Studio are an absolute mess. I think ultimately this will need be an h4 (with the title of the editing dialog being an h3), but for now you are consistent with what is happening for other component editors.